src/main.cpp

```diff
@@ -797,7 +797,7 @@ int main ( int argc, char** argv )
     // be located in the install directory of the software by the installer.
     // Here, we set the path to our application path.
     QDir ApplDir ( QApplication::applicationDirPath() );
-    pApp->addLibraryPath ( QString ( ApplDir.absolutePath() ) );
+    pAPPLICATION->addLibraryPath ( QString ( ApplDir.absolutePath() ) );
```

Comment:
Why this change? pApp is short and concise?

Reply:
> Why this change? pApp is short and concise?

In this way we are shure to call with a pointer of the correct type.
Note that this is not always the case in the current code. And though QApplication is a descendant of QCoreApplication, and has a lot of the same functions, many of these functions are NOT virtual.
This might be the cause of some GUI problems with android and iOS, since calling via a QCoreApplication pointer can skip some GUI initialization.